libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/customConfiguration.ts

```diff
@@ -1061,8 +1040,27 @@ function getYAxisTickConfiguration(
     };
 }
 
+export const getFormatterProperty = (
+    chartOptions: IChartOptions,
+    axisPropsKey: "yAxisProps" | "xAxisProps" | "secondary_yAxisProps" | "secondary_xAxisProps",
```

Comment:
Nitpick: You are passing to function `axisPropsKey` just because of check if `useCustomFormat` is true. It would be more clear to pass the function `useCustomFormat` as a parameter and do not have "dependency" on `axisPropsKey`.